config/kustomize/crd/kustomization.yaml

```diff
@@ -0,0 +1,8 @@
+# Labels to add to all resources and selectors.
+commonLabels:
+  app.kubernetes.io/instance: websphere-liberty-operator
+  app.kubernetes.io/managed-by: olm
```

Comment:
Should we have this 'managed-by' tag in here? If the operator is installed vi kustomize, then olm will not be involved at all

Reply:
It can be removed! I saw this label in all crd files so I added the label.